c/sudt_contracts.h

```diff
@@ -101,43 +159,38 @@ int transfer_to_any_sudt_gas(const uint8_t* input_src,
 
    output: []
  */
-int transfer_to_any_sudt(gw_context_t* ctx,
-                         const uint8_t* code_data,
-                         const size_t code_size,
-                         bool is_static_call,
-                         const uint8_t* input_src,
-                         const size_t input_size,
+int transfer_to_any_sudt(gw_context_t* ctx, const uint8_t* code_data,
+                         const size_t code_size, bool is_static_call,
+                         const uint8_t* input_src, const size_t input_size,
                          uint8_t** output, size_t* output_size) {
   /* Contract code hash of `SudtERC20Proxy.ContractCode`
```

Comment:
In Polyjuice v1.0.0, I think we can deprecate `SudtERC20Proxy`,
or use `SudtERC20Proxy_UserDefinedDecimals` to replace `SudtERC20Proxy` simpely.